app/controllers/BaseController.java

```diff
@@ -49,19 +49,15 @@
 
     private static ObjectMapper mapper = BridgeObjectMapper.get();
 
-    protected AuthenticationService authenticationService;
-    protected StudyService studyService;
-    protected CacheProvider cacheProvider;
-    protected BridgeConfig bridgeConfig;
+    private BridgeConfig bridgeConfig;
+    private CacheProvider cacheProvider;
 
-    @Autowired
-    public void setStudyService(StudyService studyService) {
-        this.studyService = studyService;
-    }
+    StudyService studyService;
```

Comment:
Why not private or protected? This is "package private", so another class in the package could see these (maybe I've forgotten my Java). I don't think we want that.